docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -75,6 +75,32 @@ The optional `{batch}` named query references the Batch of data under test. When
 
 :::
 
+## Dynamic Parameters
+
+Dynamic Parameters allow you to set up Expectations with automatically updating parameters based on new data, saving you the effort of maintaining Expectations as your data changes over time. Using stored validation history, GX Cloud will populate your Expectation parameters at runtime using the last `n` validation results, and your Expectation will succeed or fail based on a predetermined sensitivity to those results.
```

Comment:
I think it could be helpful to include an example of what users can do with this feature. From reading the PRD, I would say something like:

"Dynamic Parameters allow you to create Expectations whose parameters update based on new data. GX Cloud can populate new Expectation parameters at runtime using the last n validation results. For example, you can define an Expectation to validate that the maximum value within a column does not exceed 20% above a previously recorded value."

Reply:
+1